src/benchmark.cc

```diff
@@ -699,7 +743,11 @@ void RunBenchmark(const benchmark::internal::Benchmark::Instance& b,
         report.report_label = label;
         // Report the total iterations across all threads.
         report.iterations = static_cast<int64_t>(iters) * b.threads;
-        report.real_accumulated_time = real_accumulated_time;
+        if (b.use_manual_time) {
```

Comment:
I'm not sure. Usually, for my use cases (GPU workloads), the real time doesn't make sense when manual time is required, and I would be interested in having the throughput measured against the manual time.

However, it's possible that there are cases where it could be useful to have both. The console output might get a little cluttered in tabular mode, but I guess that's not a big problem.